zookeeper-server/src/main/java/org/apache/zookeeper/server/SyncRequestProcessor.java

```diff
@@ -175,9 +182,14 @@ private void flush(LinkedList<Request> toFlush)
         if (toFlush.isEmpty())
             return;
 
+        LOG.info("flush size {}", toFlush.size());
```

Comment:
Do we need this at info level? Is it useful?
Only asking

Reply:
removed